<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextBrowser" name="Input">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>11</y>
     <width>481</width>
     <height>89</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextBrowser" name="Output">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>481</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RED">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>170</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Red</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BLUE">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>230</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Blue</string>
   </property>
  </widget>
  <widget class="QPushButton" name="GREEN">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>200</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Green</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="FontSize">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>310</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="SetFontSize">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>340</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>SetFontSize</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>110</y>
     <width>102</width>
     <height>50</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QPushButton" name="FontUbuntu">
    <property name="text">
     <string>Ubuntu</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FontNanumGothic">
    <property name="text">
     <string>NanumGothic</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
